docs/user-guide/api.md

```diff
@@ -1,17 +1,36 @@
-# API Design
+# API
 
-Our API was designed with three principles in mind:
+Screwdriver APIs and the data models around them are documented via [Swagger]. This prevents out-of-date documentation, enables clients to be auto-generated, and most importantly exposes a human-readable interface.
 
-1. All interactions of user's data should be done through the API, so that
-there is a consistent interface across the tools (CLI, Web UI, etc.).
-1. Resources should be ReST-ful and operations should be atomic so that intent
-is clear and human readable.
-1. API should be versioned and self-documented, so that client code generation
-is possible.
+> **Version 4** is the current API, all links should be prefixed with `/v4`.
+
+See our [API documentation].
```

Comment:
Can we also show the endpoint `/documentation` explicitly here or in the `With Swagger` section? I feel that will be more straight forward